### DISCUSSION TITLE
Something about paper accept?

### DISCUSSION THREAD

**Discussion by havetry** [PaddlePaddle/PaddleSpatial#294] [BaiduKDDCup2022] [answered]
Dear organizers:
  I have a question about paper acceptance. As you said in your email, we have ensured that the paper will be presented as a poster in the workshop. But how is it determined whether it will be accepted and published? At the same time, is the team that accepted the invitation to the oral presentation to ensure that the article will be accepted and published?

**Comment by havetry** [comment]
@zhoujingbo

**Reply by Ethan1X** [reply]
Mark~

**Comment by wean2016** [comment]
@zhoujingbo

**Comment by zhoujingbo** [comment, marked as answer]
Hi, all;

Sorry that I missed this question. 

We have discussed with KDD Cup chairs to create a proceeding for KDD Cup, but at that time the proceeding proposal deadline for KDD 2023 has been passed.  We will put the papers on the KDD Cup website to show the credits of the winner teams. Putting the paper on the KDD Cup website is just like a pre-print, and it does not affect your future submission to other venues.

Thanks a lot

Regards
Jingbo
